src/pages/NightCream/NightCream.js

```diff
@@ -0,0 +1,101 @@
+            <div className='product-ingredients'>
+              Ingredients:&nbsp;
+              <a href="/">Oil</a>,&nbsp;
+              <a href="/">Aloe</a>,&nbsp;
+              <a href="/">Ceramides</a>
+            </div>
+            <a href="/" className="shop-button">Shop Now</a>
+          </div>
+          <div className="product-reviews">
+            <h2>Reviews</h2>
+            <div className="review">
+              <div className="review-stars">
+                <svg xmlns="http://www.w3.org/2000/svg" width="19" height="17" viewBox="0 0 19 17" fill="none">
+                  <path d="M9.31594 0L11.4075 6.43716H18.1759L12.7002 10.4155L14.7917 16.8527L9.31594 12.8743L3.84017 16.8527L5.93173 10.4155L0.455955 6.43716H7.22438L9.31594 0Z" fill="#838383" />
+                </svg>
+                <svg xmlns="http://www.w3.org/2000/svg" width="19" height="17" viewBox="0 0 19 17" fill="none">
+                  <path d="M9.31594 0L11.4075 6.43716H18.1759L12.7002 10.4155L14.7917 16.8527L9.31594 12.8743L3.84017 16.8527L5.93173 10.4155L0.455955 6.43716H7.22438L9.31594 0Z" fill="#838383" />
+                </svg>
+                <svg xmlns="http://www.w3.org/2000/svg" width="19" height="17" viewBox="0 0 19 17" fill="none">
+                  <path d="M9.31594 0L11.4075 6.43716H18.1759L12.7002 10.4155L14.7917 16.8527L9.31594 12.8743L3.84017 16.8527L5.93173 10.4155L0.455955 6.43716H7.22438L9.31594 0Z" fill="#838383" />
+                </svg>
+                <svg xmlns="http://www.w3.org/2000/svg" width="19" height="17" viewBox="0 0 19 17" fill="none">
+                  <path d="M9.31594 0L11.4075 6.43716H18.1759L12.7002 10.4155L14.7917 16.8527L9.31594 12.8743L3.84017 16.8527L5.93173 10.4155L0.455955 6.43716H7.22438L9.31594 0Z" fill="#838383" />
+                </svg>
+                <svg xmlns="http://www.w3.org/2000/svg" width="19" height="17" viewBox="0 0 19 17" fill="none">
+                  <path d="M9.31594 0L11.4075 6.43716H18.1759L12.7002 10.4155L14.7917 16.8527L9.31594 12.8743L3.84017 16.8527L5.93173 10.4155L0.455955 6.43716H7.22438L9.31594 0Z" fill="#838383" />
+                </svg>
+              </div>
+              <div className="review-text">
+                "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore”
+              </div>
+              <div className="review-author">
+                <span>
+                  <svg xmlns="http://www.w3.org/2000/svg" width="50" height="51" viewBox="0 0 50 51" fill="none">
+                    <circle cx="25" cy="25.6318" r="25" fill="#838383" />
+                  </svg>
+                </span>
+                <span>Person Name</span>
+              </div>
+            </div>
+            <div className="review">
+              <div className="review-stars">
```

Comment:
I would try to store the number of stars that the review is giving the product as a variable and render a star that many of times with some loop logic. I'm assuming that the client will only be showing 5-star reviews on the site, but it's good practice!